model/state_variables.py

```diff
@@ -35,6 +35,9 @@ class StateVariables(TypedDict):
     reserve_balance: Balance
     mento_buckets: Dict[MentoExchange, MentoBuckets]
     market_price: Dict[Pair, float]
+    reserve_balance_in_usd: float
```

Comment:
floating_supply_stables_in_usd seams to be missing here?

Reply:
jup